build/frontend.asset.php

```diff
@@ -1 +1 @@
-<?php return array('dependencies' => array('wp-element', 'wp-polyfill'), 'version' => '889e873949c9ad8654a574a976c91cc5');
\ No newline at end of file
+<?php return array('dependencies' => array('wp-element', 'wp-polyfill'), 'version' => '77379583903f779f03cdbca8a0c3b6dc');
```

Comment:
I might have forget to mention this, but `/build` should be gitignored.

Reply:
Thanks for pointing this out. I added `build` to the `.gitignore` file and ran the following command to remove the `build` folder from the remote repo:

```
$ git rm -r --cached build
$ git commit -m "Remove build folder from repository"
$ git push origin trunk -f
```